updatecli/update_third_party_files.yaml

```diff
@@ -107,7 +127,38 @@ targets:
         - name: "FILES"
           value: "pkg/api/resource/amount.go pkg/api/resource/quantity.go pkg/api/resource/math.go \
                   pkg/api/resource/scale_int.go pkg/api/resource/suffix.go"
+  updateCelGoVersion:
```

Comment:
I have a doubt about this, isn't the update handled by our bots anyway?

Reply:
You're right. Renovate bot updates the cel-go. I'll remove this from the PR.